tcn_hpl/models/ptg_module.py

```diff
@@ -123,9 +122,6 @@ def __init__(
         self.validation_step_outputs_source_vid = []
         self.validation_step_outputs_source_frame = []
 
-        hydra_cfg = HydraConfig.get()
-        self.output_dir = hydra_cfg['runtime']['output_dir']
-
         # Load val vidoes
         vid_list_file_val = f"{self.hparams.data_dir}/splits/val.split1.bundle"
         with open(vid_list_file_val, "r") as val_f:
```

Comment:
This will still fail during inference: I don't expect any of this to exist during pure inference mode so the file opening will fail.

Reply:
updated